### PR TITLE
Feature/view submission

### PR DESCRIPTION
**Pull request by TimCsaky**
I've made a start.
- added call from frontend to api to get a single IPC plan (currently returns all for some reason)
- added a frontend route to bring up the view on a new page

### CODE REVIEW

File: app/frontend/src/components/admin/SubmissionsTable.vue [hunk 1]
```diff
@@ -25,19 +25,26 @@
         :search="search"
         :loading="loading"
         loading-text="Loading... Please wait"
-        item-key="id"
+        item-key="confirmationId"
```
Comment (TimCsaky) [inline app/frontend/src/components/admin/SubmissionsTable.vue:28]: this fixes the issue Matt just mentioned where all rows were expanding

File: app/frontend/src/components/admin/SubmissionsTable.vue [hunk 2]
```diff
@@ -25,19 +25,26 @@
...
         class="ipc-table"
         :expanded.sync="expanded"
         show-expand
       >
+        <!-- view individual form submission -->
+        <template v-slot:item.ipcPlanId="{ ipcPlanId }">
+          <router-link :to="{ name: 'Submission', params: { id: ipcPlanId } }"><v-icon>remove_red_eye</v-icon></router-link>
```
Comment (TimCsaky) [inline app/frontend/src/components/admin/SubmissionsTable.vue:35]: i'm guessing this is where i pass the ipcPlanId but I need to fix this

File: app/frontend/src/views/Submission.vue [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  data() {
+    return {
+      ipcPlanId: ''
+    };
+  },
+  methods: {
+    // get table data from frontend service layer
+    getData() {
+      ipcService
+        .getIPCContent('7315ba58-fb9d-4e88-8896-22180763355b')
```
Comment (TimCsaky) [inline app/frontend/src/views/Submission.vue:94]: this is where i send the ipcPlanId to the backend. I was testing with a value from my database.
the console.log shows me i am getting all form submissions

Comment (loneil) [reply app/frontend/src/views/Submission.vue:94]: In order for any of those components to render they'll need the state updated (they pull all fields computed from the Vuex store).
So this isn't where the call to get the form would be. It should dispatch an action on mounted that gets the form and then mutates the vuex store to the state it needs.

Comment (TimCsaky) [reply app/frontend/src/views/Submission.vue:94]: ok. thanks. Appreciate the input. to be honest i'm not sure i understand this. i'll try and take another look at this tonight

Comment (loneil) [reply app/frontend/src/views/Submission.vue:94]: Ok no prob. I'll give it a think too. TBH I'm not 100% on how it's all going to fit together fetching store state on the admin side. I'm fairly certain we just need an action to fetch a specific form by ID as you've got and then set the existing objects in the store with those results via the existing mutations.
I'm a bit concerned about what to do with the store when switching from one form to another so I think we'll need a way to clear it as well, or rock-solid error handling.
Might need to switch over to dynamic loading the same way we do in some modules in GETOK

**Pull request closed without merging**